src/app/kbv/controllers/question.js

```diff
@@ -10,6 +10,9 @@ const { API } = require("../../../lib/config");
 
 class QuestionController extends BaseController {
   configure(req, res, next) {
+    if (!req.session.question) {
+      return next(new Error("Current session has no Question to configure."));
```

Comment:
So when we throw these errors are they easy to find in splunk/dynatrace or would we also find a log helpful?

Reply:
Yes, this should appear in the logs as is, what we don't have is the informational logs up to what errors.